models/docs.md

```diff
@@ -1931,3 +1931,51 @@ List of included roles.
 {% docs sys_user_group_roles %}
 List of associated user group roles.
 {% enddocs %}
+
+{% docs sys_choice %}
+Staging table containing all records relating to system choices. System choices represent predefined options for fields in various ServiceNow tables.
```

Comment:
This is the definition of the source table. I recommend removing the staging wording from this description.

```suggestion
Table containing all records relating to system choices. System choices represent predefined options for fields in various ServiceNow tables.
```